owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/RemoteShare.java

```diff
@@ -1,5 +1,5 @@
 /* ownCloud Android Library is available under MIT license
- *   Copyright (C) 2017 ownCloud GmbH.
+ *   Copyright (C) 2019 ownCloud GmbH.
```

Comment:
This header makes only afford, I would remove it complete.
There is a global license file !
Same to `* @author` it only to praise the initial commiter (yourself) 
A git history has more weight and has more humility

@michaelstingl maybe you should discuss this